doc/ChangeLog

```diff
@@ -1,4 +1,125 @@
 ===============================================================
+Tag name: ctsm5.4.000
+Originator(s): slevis (Samuel Levis,UCAR/TSS), ekluzek (Erik Kluzek,UCAR/TSS)
```

Comment:
This could be you and I as here, or multiple, which is what we did for ctsm5.3.0.